.github/workflows/create-issue-with-branch.yml

```
@@ -0,0 +1,55 @@
+        run: |
+          # 레이블이 없는 경우 "other"로 기본값 설정
+          if [ -z "${{ toJson(github.event.issue.labels) }}" ]; then
+            echo "label=other" >> $GITHUB_ENV
+          else
+            LABEL=$(echo '${{ toJson(github.event.issue.labels) }}' | jq -r '.[0].name')
+            echo "label=${LABEL}" >> $GITHUB_ENV
+          fi
+
+      - name: Extract Keyword
+        id: extract-keyword
+        run: |
```

actionlint (1.7.7)

"github.event.issue.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)